google/ggrc-core
# GGRC-8363 Replace can-map with can-define-map for advanced-search's components

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #10467


# Issue description

Actually can-map, can-list are used. can-map, can-list are legacy and the alternatives in latest versions of CanJS are DefineMap, DefineList.

# Steps to test the changes

Open the APP and check Advanced \ Global searches, Object Mapper, Bulk Verify, Bulk Complete, Assessment template cloner.

Also, need to check Multiselect dropdown and tree-view filter

# Sanity checklist




# PR Review checklist
